lib/model.js

```diff
@@ -312,7 +299,7 @@ const _create = (data, definition) => {
     debug(`remove from ${collection} successful`);
     if ('function' === typeof this.afterRemove) {
       const afterRemove = this.afterRemove(options);
-      if (options.waitAfterRemove) { await afterRemove; }
+      if (options?.waitAfterRemove) { await afterRemove; }
```

Comment:
you could default options = {} instead of using ?

Reply:
done